pkgs/development/python-modules/itanium_demangler/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "itanium_demangler";
+  version = "1.0";
```

the version should really match a version that the upstream repo has actually cut. Since this repo just has a few commits but no tags/releases, you would want to use "unstable-${date of commit}".
```suggestion
  version = "unstable-2018-10-28";
```
However, since they pushed it to pypi at version 1.0, im fine with this as well:
```suggestion
  version = "1.0"; # pulled from pypi version
```
just to describe why it's different from the expected norm